docusaurus.config.js

```diff
@@ -75,7 +178,13 @@ const config = {
       },
       colorMode: {
         respectPrefersColorScheme: true,
-      }
+      },
+      algolia: {
+        appId: 'someAppId',
+        apiKey: 'someApiKye',
+        indexName: 'someIndexName',
+        contextualSearch: true,
```

Comment:
Please remove it for now if we don't have the config

Reply:
It's not harm anything it simple enables this search bar thats it.
<img width="1643" alt="image" src="https://user-images.githubusercontent.com/14016055/187444188-fd5cca8a-7d2f-441b-9ed6-85eb5ae036d8.png">